src/App.js

```diff
@@ -0,0 +1,272 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react'
+import './App.css';
+import { BrowserRouter as Router, Route, Link, Redirect, Switch } from 'react-router-dom'
+import axios from "axios";
+import { AuthService } from './services/Auth.service';
+import Operations from './components/Operations';
+import Transactions from './components/Transactions';
+import Breakdown from './components/Breakdown';
+import Login from './components/Login';
+import 'antd/dist/antd.css';
+import { Layout, Menu, PageHeader, Typography, Space, message } from 'antd';
+import {
+  FileAddOutlined,
+  MoneyCollectOutlined,
+  MenuOutlined,
+  LogoutOutlined
+} from '@ant-design/icons';
+
+const { Sider, Content } = Layout;
+const { Text } = Typography;
+const authService = new AuthService();
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      transactions: [],
+      aggregatedTransactions: [],
+      collapsed: false,
+      redirect: false,
+      loggedIn: authService.isLoggedIn(),
+      pageTitle: 'My Transactions'
+    }
+  }
+
+  setData = async (token, del = false) => {
+    try {
+      const config = {
+        headers: { Authorization: `Bearer ${token}` }
+      };
+      const resp = await axios.get('/transactions', config);
```

Comment:
Will be a good idea (as a continuation to the server architecture), to extract all API calls to a service (something like: `HttpServicez)

Reply:
It is a good idea, I'll add it to my todo list.